### PR TITLE
Mobile layout!

### PR DESCRIPTION
**Pull request by koddsson**
I tried not to mess with the desktop version too much. This also adds tabzilla like how it's described in the tabzilla docs and generally tries to follow the sandstone best practices.

Give it a whirl and tell me what you think.

mobile
desktop

### CODE REVIEW

**Comment by peterbe** [comment]
@schalkneethling wanna take a look at this one?

**Comment by schalkneethling** [comment]
@peterbe yeah sure, will take a look. Thanks for the PR @koddsson

File: app/static/css/contribute/contribute.less [hunk 1]
```diff
@@ -29,6 +27,9 @@ h1 {
 }
 h2 {
     font-size: 2rem;
+    @media all and (max-width: 768px) {
+        display: none;
```
Comment (peterbe) [inline app/static/css/contribute/contribute.less]: Why should the H2's be hidden? That seems a bit unfair.

**Comment by peterbe** [comment]
I'm r+ but I still question why the h2 tags should disappear when on a media query. It feels just "off" that content should disappear. It'd be a different thing if it was a big unnecessary but sexy image or something. 

Also, can you fold in these? https://github.com/koddsson/contribute.json/pull/2

**Comment by koddsson** [comment]
@peterbe The double Mozilla icon felt a little repetitive to me in the mobile view with the addition of TabZilla. It's no problem adding it back in if you want :)

I just noticed that hiding all `h2` elements would also hide some 'Use cases' headings so I pushed a better fix.

contribute json after
contribute json before

**Comment by peterbe** [comment]
A ha! I didn't even notice that the logo disappeared with the h2 media query. Seems it's worked out now. This looks awesome. I'll push it out on paas now.

**Comment by koddsson** [comment]
Awesome :)

**Pull request merged**